ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1506,9 +1569,99 @@ class RangeSelector {
         const {
             buttonPosition,
             inputPosition,
-            verticalAlign
+            verticalAlign,
+            inputBoxHeight,
+            inputBoxBorderColor
         } = options;
 
+        this.maxDateBox?.attr({
+            height: inputBoxHeight
+        });
+
+        this.minDateBox?.attr({
+            height: inputBoxHeight
+        });
+
+        if (!chart.styledMode) {
+            this.maxDateBox?.attr({
+                stroke: inputBoxBorderColor
+            });
+
+            this.minDateBox?.attr({
+                stroke: inputBoxBorderColor
+            });
+        }
```

Comment:
The height and stroke attribs shouldn't be part of `alignElements` method. Please, keep this method clean and short

Reply:
The stroke can be elsewhere, but the height may have an impact to the rangeSelector alignment, so I can't move it.